rules/windows/c2_grey_list_beacon_sequence.toml

```diff
@@ -0,0 +1,88 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies network activity from unexpected system applications. This may indicate adversarial activity as these
+applications are often leveraged by adversaries to execute code and evade detection.
+"""
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Unusual Process Network Connection: Sequence"
+risk_score = 21
+rule_id = "1fe3b299-fbb5-4657-a937-1d746f2c711a"
+severity = "medium"
+tags = ["from-eql-rules"]
+type = "eql"
+
+query = '''
+sequence by process.entity_id, process.pid, process.name with maxspan=5m
+  [process where event.type in ("start", "process_started") and
+
+     // known applocker bypasses
+     process.name in ("bginfo.exe",
+                      "cdb.exe",
+                      "control.exe",
+                      "cmstp.exe",
+                      "csi.exe",
+                      "dnx.exe",
+                      "fsi.exe",
+                      "ieexec.exe",
+                      "iexpress.exe",
+                      "installutil.exe",
+                      "Microsoft.Workflow.Compiler.exe",
+                      "msbuild.exe",
+                      "msdt.exe",
+                      "mshta.exe",
+                      "msiexec.exe",
+                      "msxsl.exe",
+                      "odbcconf.exe",
+                      "rcsi.exe",
+                      "regsvr32.exe",
+                      "xwizard.exe")]
+  [network where event.type == "connection"]
```

Comment:
Related to https://github.com/elastic/detection-rules/pull/304/files#r490017645 - is there a performance gain by specifying constraining fields in the sequence fields or additionally specifying the full list of `process.name`s here too (or both)

Reply:
there is a recent optimization for 7.10 that applies when there are overlapping join keys, but I think for now we should be explicit and copy/paste the `process.name` check. this will also help out for any users of 7.9 that try this rule with the API manually